homepage.css

```diff
@@ -100,6 +106,8 @@ ul li:hover ul{
 	text-align: center;
 	color:white;
 	height:50px;
+	font-family: 'Open Sans', sans-serif;
```

Comment:
10/09/2021: Changing the fonts style for the footer to be Open Sans, also the font weight.